bin/civihr-install

```diff
@@ -0,0 +1,820 @@
+#!/bin/bash
```

Comment:
Why is the script name civihr-install and not civihr-install.sh? I think it's always a good practice to keep the extension. If the user doesn't want it, they can always removed it